bears/general/SpaceConsistencyBear.py

```diff
@@ -14,7 +14,9 @@ def run(self,
             tab_width: int=SpacingHelper.DEFAULT_TAB_WIDTH,
             enforce_newline_at_EOF: bool=True):
         '''
-        Checks the space consistency for each line.
+        Check and correct spacing for all textual data. This includes usage of
+        tabs vs. spaces, trailing whitespace and (missing) newlines before
+        the end of the file.
```

Comment:
do you think it's worth adding the link from stackoverflow regarding newlines before the EOF here?